docs/how-to-use-minio-as-laravel-file-storage.md

```diff
@@ -0,0 +1,105 @@
+# How to use Minio Server as [Laravel](https://laravel.com) Custom File Storage 
+
+`Laravel` has a customable file storage system with ability to create custom drivers for it. In this recipe we will implement a custom file system driver to use Minio server for managing files.
```

Comment:
typo: it should be customizable instead of customable

Reply:
Done.